indico/modules/events/registration/api.py

```diff
@@ -67,3 +69,8 @@ def _process_args(self):
 
     def _process_GET(self):
         return jsonify(registrants=build_registrations_api_data(self.event))
+
+
+class RHAPIRegistrationForms(RHProtectedEventBase):
+    def _process(self):
+        return jsonify(RegistrationFormSchema().dump(self.event.registration_forms, many=True))
```

Comment:
```suggestion
        return RegistrationFormSchema(many=True).jsonify(self.event.registration_forms)
```

Reply:
Isn't it the same thing?

Reply:
yes, it's just a shortcut to do it:

https://github.com/marshmallow-code/flask-marshmallow/blob/92d2c20ab8da5e669c7327f39c1f7a94f021ca3d/src/flask_marshmallow/schema.py#L16